gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiPlansResource_CloseTest.java

```diff
@@ -15,54 +15,117 @@
  */
 package io.gravitee.rest.api.management.v2.rest.resource.api;
 
-import static io.gravitee.common.http.HttpStatusCode.FORBIDDEN_403;
-import static io.gravitee.common.http.HttpStatusCode.OK_200;
+import static io.gravitee.common.http.HttpStatusCode.*;
 import static org.junit.Assert.assertEquals;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.*;
 
+import fixtures.PlanFixtures;
+import io.gravitee.rest.api.management.v2.rest.model.Error;
+import io.gravitee.rest.api.management.v2.rest.model.PlanV2;
+import io.gravitee.rest.api.management.v2.rest.model.PlanV4;
 import io.gravitee.rest.api.model.permissions.RolePermission;
-import io.gravitee.rest.api.model.v4.api.ApiEntity;
+import io.gravitee.rest.api.model.permissions.RolePermissionAction;
 import io.gravitee.rest.api.model.v4.plan.PlanEntity;
 import io.gravitee.rest.api.service.common.GraviteeContext;
+import io.gravitee.rest.api.service.exceptions.PlanNotFoundException;
+import io.gravitee.rest.api.service.v4.PlanSearchService;
 import jakarta.ws.rs.client.Entity;
 import jakarta.ws.rs.core.Response;
 import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 
 public class ApiPlansResource_CloseTest extends ApiPlansResourceTest {
 
     @Override
     protected String contextPath() {
-        return "/environments/" + ENVIRONMENT + "/apis/" + API + "/plans";
+        return "/environments/" + ENVIRONMENT + "/apis/" + API + "/plans" + "/" + PLAN + "/_close";
     }
 
+    @Autowired
+    private PlanSearchService planSearchService;
+
     @Test
-    public void shouldCloseApiPlan() {
-        ApiEntity api = getApi();
+    public void should_return_404_if_not_found() {
+        when(planSearchService.findById(GraviteeContext.getExecutionContext(), PLAN)).thenThrow(new PlanNotFoundException(PLAN));
 
-        PlanEntity existingPlan = new PlanEntity();
-        existingPlan.setName(PLAN);
-        existingPlan.setApiId(API);
+        final Response response = rootTarget().request().post(Entity.text(""));
```

Comment:
question: would it be better to use `null` when there is no body for a post request ?
```suggestion
        final Response response = rootTarget().request().post(null);
```